test/integration/login_flows_test.rb

```diff
@@ -50,7 +50,7 @@ class LoginFlowsTest < ActionDispatch::IntegrationTest
 
     assert_select '.notification', text: 'Account was successfully created'
 
-    get account_path
```

Comment:
this is a breaking change; meaning we lost a pre-existent root; 
You should not change the test, but add again that route, which will be just an alias to /users/:username if not possible put again the account action.

Reply:
No, the '/account' route will keep working without a problem (see the screenshot)
the problem in the tests was related to the status of the response instead of getting 200, the rails server since the endpoint doesn't match the name of the controller is concidering it as a redirection 
![image](https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/61744974/e0084ff6-3d4d-4d25-bbaa-0eecaf3fb9f5)

and this route '/account' really need to be deleted cause it's used only to display the account page of the user and not used in any of the operations like update ...